x-pack/plugins/logs_shared/kibana.jsonc

```diff
@@ -8,10 +8,9 @@
     "server": true,
     "browser": true,
     "configPath": ["xpack", "logs_shared"],
-    "requiredPlugins": ["data", "dataViews", "usageCollection", "observabilityShared"],
+    "requiredPlugins": ["data", "dataViews", "usageCollection", "observabilityShared", "observabilityAIAssistant"],
     "optionalPlugins": ["observability"],
```

Comment:
@weltenwort Do we need `observability` to be optional still?

Reply:
Looks like our `public` `setup` still depends on it. It even seems to require it, so I'm not sure why it was only optional before. :shrug:

Reply:
Sorry, I looked in the wrong place. The `logs_shared` plugin apparently only depended on `observability` for copilot stuff, so it's not necessary anymore. Thanks a lot for checking!